### PR TITLE
Made changes to collect new metrics disk read/write and other database related metics

### PR DESCRIPTION
**Pull request by madhutadiparthi4ibm**
https://instana.kanbanize.com/ctrl_board/202/cards/160618
Made changes to collect new metrics mentioned below from Informix.
db.disk.read.count
db.disk.write.count
db.database.log.enabled
db.database.buff.log.enabled
db.database.ansi.complaint
db.database.nls.enabled
db.database.case.insensitive

### CODE REVIEW

**Comment by Gaozhenz** [comment]
Hi @BikramDebnath1  
@madhutadiparthi4ibm implemented the metrics collection in this PR. Please check, thanks.

File: rdb/src/main/java/com/instana/dc/rdb/impl/informix/InformixUtil.java [hunk 1]
```diff
@@ -39,6 +39,14 @@ private InformixUtil() {
     public static final String TABLESPACE_UTILIZATION_SQL = "select case WHEN (pt.nptotal > 0) THEN ((pt.npused) /pt.nptotal) * 100 ELSE 0 END AS table_utilization, tabname FROM sysmaster:sysptnhdr pt INNER JOIN sysmaster:systabnames tn ON tn.partnum = pt.partnum where (tn.dbsname in ('instana')) order by tabname desc Limit 40;";
     public static final String TABLESPACE_MAX_SQL = "SELECT(pt.nptotal * pt.pagesize)  * 1024 AS total_kb, tabname FROM sysmaster:sysptnhdr pt INNER JOIN sysmaster:systabnames tn ON tn.partnum = pt.partnum where (tn.dbsname in ('instana')) order by tabname desc Limit 40;";
 
+    public static final String DB_DISK_WRITE_COUNT_SQL = "SELECT value From sysprofile where name = 'dskwrites'";
+    public static final String DB_DISK_READ_COUNT_SQL = "SELECT value From sysprofile where name = 'dskreads'";
+
+    public static final String DB_DATABASE_LOG_ENABLED_SQL = "SELECT is_logging FROM sysdatabases";
+    public static final String DB_DATABASE_BUFF_LOG_ENABLED_SQL = "SELECT is_buff_log FROM sysdatabases";
```
Comment (BikramDebnath1) [inline rdb/src/main/java/com/instana/dc/rdb/impl/informix/InformixUtil.java:46]: Please have a look at the semantic over [here](https://github.com/instana/otel-dc/blob/main/docs/semconv/database.md#settings-metrics)
each metrics has a mandatory attribute i.e. `database_name`

File: rdb/src/main/java/com/instana/dc/rdb/impl/informix/InformixDc.java [hunk 2]
```diff
@@ -249,6 +266,15 @@ private void shortPollingInterval() {
         getRawMetric(DB_IO_READ_RATE_NAME).setValue((Number) metricCollector.collectMetrics(DB_IO_READ_RATE_NAME));
         getRawMetric(DB_IO_WRITE_RATE_NAME).setValue((Number) metricCollector.collectMetrics(DB_IO_WRITE_RATE_NAME));
         getRawMetric(DB_MEM_UTILIZATION_NAME).setValue((Number) metricCollector.collectMetrics(DB_MEM_UTILIZATION_NAME));
+
+        getRawMetric(DB_DISK_WRITE_COUNT_NAME).setValue((Number) metricCollector.collectMetrics(DB_DISK_WRITE_COUNT_NAME));
+        getRawMetric(DB_DISK_READ_COUNT_NAME).setValue((Number) metricCollector.collectMetrics(DB_DISK_READ_COUNT_NAME));
```
[the remark this answers is not in the document]
Comment (madhutadiparthi4ibm) [reply rdb/src/main/java/com/instana/dc/rdb/impl/informix/InformixDc.java:271]: Addressed suggested changes

File: rdb/src/main/java/com/instana/dc/rdb/impl/informix/InformixUtil.java [hunk 3]
```diff
@@ -39,6 +39,11 @@ private InformixUtil() {
     public static final String TABLESPACE_UTILIZATION_SQL = "select case WHEN (pt.nptotal > 0) THEN ((pt.npused) /pt.nptotal) * 100 ELSE 0 END AS table_utilization, tabname FROM sysmaster:sysptnhdr pt INNER JOIN sysmaster:systabnames tn ON tn.partnum = pt.partnum where (tn.dbsname in ('instana')) order by tabname desc Limit 40;";
     public static final String TABLESPACE_MAX_SQL = "SELECT(pt.nptotal * pt.pagesize)  * 1024 AS total_kb, tabname FROM sysmaster:sysptnhdr pt INNER JOIN sysmaster:systabnames tn ON tn.partnum = pt.partnum where (tn.dbsname in ('instana')) order by tabname desc Limit 40;";
 
+    public static final String DB_DATABASE_LOG_ENABLED_SQL = "SELECT is_logging, name FROM sysdatabases";
```
Comment (BikramDebnath1) [inline rdb/src/main/java/com/instana/dc/rdb/impl/informix/InformixUtil.java:42]:
```suggestion
    public static final String DB_DATABASE_LOG_ENABLED_SQL = "SELECT is_logging, name as database_name FROM sysdatabases";
```

just to have a clear understanding what does the attribute `name` mean over here. 
Same is applicable to the other queries also.

**Review by BikramDebnath1 — APPROVED** [review]
LGTM

**Pull request merged**